Uni_life_tracker/app/src/main/res/layout/activity_add_event.xml

```diff
@@ -0,0 +1,267 @@
+
+        <ImageView
+                android:id="@+id/iv_save"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:onClick="ClickSave"
+                android:src="@drawable/ic_done"
+                android:contentDescription="@string/save"/>
+    </LinearLayout>
+
+    <!-- Event Type Selection -->
+    <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:orientation="horizontal"
+            android:gravity="center"
+            android:padding="5dp">
+
+        <!-- Assessment -->
+        <TextView
+                android:id="@+id/tv_assessment"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                app:drawableTopCompat="@drawable/ic_assessment"
+                android:text="@string/assessment"
+                android:textColor="@color/black"
+                android:fontFamily="@font/roboto_mono_bold"
+                android:textAlignment="center"
+                android:padding="15dp"
+                android:onClick="ClickAssessment"/>
+
+        <!-- Deadline -->
+        <TextView
+                android:id="@+id/tv_deadline"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                app:drawableTopCompat="@drawable/ic_deadline"
+                android:text="@string/deadline"
+                android:textColor="@color/black"
+                android:fontFamily="@font/roboto_mono_bold"
+                android:textAlignment="center"
+                android:padding="15dp"
+                android:onClick="ClickDeadline"/>
+
+        <!-- Class Time -->
+        <TextView
+                android:id="@+id/tv_class_time"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                app:drawableTopCompat="@drawable/ic_class_time"
+                android:text="@string/class_time"
+                android:textColor="@color/black"
+                android:fontFamily="@font/roboto_mono_bold"
+                android:textAlignment="center"
+                android:padding="15dp"
+                android:onClick="ClickClassTime"/>
+
+        <!-- Study Session -->
+        <TextView
+                android:id="@+id/tv_study_session"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                app:drawableTopCompat="@drawable/ic_study_session"
+                android:text="@string/study_session"
+                android:textColor="@color/black"
+                android:fontFamily="@font/roboto_mono_bold"
+                android:textAlignment="center"
+                android:padding="15dp"
+                android:onClick="ClickStudySession"/>
+    </LinearLayout>
+
+    <!-- Add Event Title -->
+    <EditText
```

Comment:
A visual thing: but would it be possible to make the event title line a little short on both ends so it lines up with the beginning of assessment and the end of study session?